src/win32/win32vulkanvideo.cpp

```diff
@@ -49,7 +50,7 @@ bool I_GetVulkanPlatformExtensions(unsigned int *count, const char **names)
 	else
 	{
 		const bool result = *count >= extensionCount;
-		*count = min(*count, extensionCount);
+		*count = std::min<unsigned int>(*count, extensionCount);
```

Comment:
Would I be a little too out of line to suggest using the templates that dpJudas suggested first? It's basically an uppercase `MIN()` function.

Reply:
As i said i already had fixed it myself first when i read his post, std::min is also used a lot and it would need another include anyway. He also suggested using ifdef/undef. Why should i change it?

Reply:
The homegrown macros should be phased out in favor of official language features.

Reply:
Fair enough.